# Goal
uint8 SET_GOAL_POSITION = 0
uint8 FOLLOW_PATTERN = 1
uint8 TRACKING = 2
uint8 TERMINATE_SCAN = 3

string[] name
float64[] position
float64[] velocity

string pattern

float64 errorPan
float64 errorTilt

uint8 command
---
# Result

bool success